sdkproject/Assets/Mapbox/Unity/MeshGeneration/Components/FeatureBehaviour.cs

```diff
@@ -7,14 +7,36 @@ namespace Mapbox.Unity.MeshGeneration.Components
 
 	public class FeatureBehaviour : MonoBehaviour
 	{
-		public Transform Transform { get; set; }
+		public VectorEntity VectorEntity;
+		public Transform Transform;
 		public VectorFeatureUnity Data;
 
 		public void ShowDebugData()
 		{
 			DataString = string.Join("\r\n", Data.Properties.Select(x => x.Key + " - " + x.Value.ToString()).ToArray());
 		}
 
+		public void ShowDataPoints()
+		{
+			foreach (var item in VectorEntity.Feature.Points)
```

Comment:
Everything looks good to me. The only thing that I want to point out is generally it's best if we use for loop instead of foreach loop. But, the performance impact isn't that much, and considering that this is just used for debugging in this case, we can ignore it.